# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1368733568 14400
#      Thu May 16 15:46:08 2013 -0400
# Node ID 784c765225ba7a254fb692c4e2ea67210998c4ba
# Parent  dddbb5dcbf6efa67be9bb72389a75dd09f2726b2
* contributors.in: Include Risto Vanhanen in the list.

diff --git a/doc/interpreter/contributors.in b/doc/interpreter/contributors.in
--- a/doc/interpreter/contributors.in
+++ b/doc/interpreter/contributors.in
@@ -280,16 +280,17 @@ Christophe Tournery
 Thomas Treichl
 Karsten Trulsen
 Frederick Umminger
 Utkarsh Upadhyay
 Daniel Wagenaar
 Stefan van der Walt
 Peter Van Wieren
 James R. Van Zandt
+Risto Vanhanen
 Gregory Vanuxem
 Ivana Varekova
 Thomas Walter
 Andreas Weber
 Olaf Weber
 Thomas Weber
 Rik Wehbring
 Bob Weigel
